### ISSUE TITLE
Any plans on adding to npm?

### ISSUE THREAD

**Issue by jakehm** [openplans/Leaflet.AnimatedMarker#44]
This is the best way to animated a marker on leaflet that I've found.  It would be easier to implement if there was an npm package.

And this requires jquery right?  Add that the to the readme at least.

**Comment by Abbe98** [comment]
No it does not require jQuery.

**Comment by jakehm** [comment]
Because he links to it in the index.html for some reason.

**Comment by Abbe98** [comment]
Yes it's used in the demo(`demo.js`) file for loops etc. If you take a look at the plugin's source you won't find any dependency on jQuery.

**Comment by paullryan** [comment]
All thats needed to publish this is changing the name in package.json to **leaflet.animatedmarker** it's required to be all lowercase.
